bittide-instances/src/Bittide/Instances/Hitl/SwitchDemo.hs

```diff
@@ -81,6 +81,7 @@ import Protocols.Wishbone
 import System.FilePath ((</>))
 import VexRiscv (DumpVcd (..), Jtag, JtagIn (..), JtagOut (..))
 
+import Bittide.ClockControl.StabilityChecker (StabilityIndication (..))
```

Comment:
Hopefully fourmolu will support it soon, but: `qualified` imports go last as a group. Could you move this import up?